DDI.csv

```diff
@@ -1,285 +1,292 @@
+14008,Joeres 1987,https://pubmed.ncbi.nlm.nih.gov/3628469/,Caffeine + Mexiletine,Caffeine,Mexiletine,PO,PO,,Plasma,2.83,,,,1.89,,,,0,inf,0,inf,h,,,200,,mg,PO,,,,,,,,,,,,,
+14010,Kusumoto 2001,https://pubmed.ncbi.nlm.nih.gov/11240973/,Mexiletine + Fluvoxamine,Mexiletine,Fluvoxamine,PO,PO,,Plasma,1.55,,,,1.16,,,,168,inf,168,inf,h,,,50,,mg,PO,,,,,,,,,,,,,
+15001,Yao 2003,https://pubmed.ncbi.nlm.nih.gov/12695344/,With 37.5mg Fluvoxamine,S-Mephenytoin,Fluvoxamine,PO,PO,,Plasma,4.64,,,,2.12,,,,176,inf,176,inf,h,,,27.5,,mg,PO,,,,,,,,,,,,,
+15002,Yao 2003,https://pubmed.ncbi.nlm.nih.gov/12695344/,With 62.5mg Fluvoxamine,S-Mephenytoin,Fluvoxamine,PO,PO,,Plasma,6.7,,,,2.4,,,,176,inf,176,inf,h,,,45.8,,mg,PO,,,,,,,,,,,,,
+15003,Yao 2003,https://pubmed.ncbi.nlm.nih.gov/12695344/,With 87.5mg Fluvoxamine,S-Mephenytoin,Fluvoxamine,PO,PO,,Plasma,9.89,,,,2.42,,,,176,inf,176,inf,h,,,64.1,,mg,PO,,,,,,,,,,,,,
+16062,Bailey 1996,https://pubmed.ncbi.nlm.nih.gov/8689808/,10mg-PO-ER tablet-DDI,Felodipine,Erythromycin,PO,PO,,Plasma,2.592986331,Geom. mean,1.256999328,Geom. SD,2.288864202,Geom. mean,1.174586181,Geom. SD,0,24,24,48,h,,,250,,mg,PO,(S0-T6-R2),h,four times a day for 2 days,,,Tablet,,,,Fasted,10,4,
+16063,Jalava 1997,https://pubmed.ncbi.nlm.nih.gov/9129558/,5mg-PO-ER tablet-DDI,Felodipine,Itraconazole,PO,PO,,Plasma,6.335849057,Approx. avg,,,7.75,Approx. avg,,,0,inf,73,inf,h,,,200,,mg,PO,(S0-T24-R4),h,,,,Sporanox,,,,,,,
+16065,Lee 2021,https://pubmed.ncbi.nlm.nih.gov/33348240/,"clarithromycin DDI, treatment group",Sildenafil,Clarithromycin,PO,PO,,Plasma,2.276642774,Approx. avg,,,1.715718811,Approx. avg,,,0,inf,96,inf,h,,,250,,mg,PO,48 + 72 + 84 + 96,h,"SD on D3, BID on D4, SD on D5",,,Tablet,,,,,,,
+16066,Lee 2021,https://pubmed.ncbi.nlm.nih.gov/33348240/,"itraconazole DDI, treatment group",Sildenafil,Itraconazole,PO,PO,,Plasma,2.499420416,Approx. avg,,,1.433340346,Approx. avg,,,0,inf,96,inf,h,,,100,,mg,PO,48 + 72 + 84 + 96,h,"SD on D3, BID on D4, SD on D5",,,Tablet,,,,,,,
+16067,Hedaya 2006,https://pubmed.ncbi.nlm.nih.gov/16372380/,"clarithromycin DDI, treatment group",Sildenafil,Clarithromycin,PO,PO,,Plasma,2.280739161,Approx. avg,,,2.418118467,Approx. avg,,,0,inf,2,inf,h,,,50,,mg,PO,2,h,SD,,,Tablet,,,,Fasted,8,3,
+16068,Hesse 2005,https://pubmed.ncbi.nlm.nih.gov/16282844/,"fluvoxamine DDI, control group",Sildenafil,Fluvoxamine,PO,PO,,Plasma,1.403940887,Approx. avg,,,1.303317536,Approx. avg,,,0,inf,240,inf,h,,,"50, 100",,mg,PO,"50mg: S0-T24-R3, 100mg: S72-T24-R7",h,OD,,,Coated tablet,,,,,,,
+16073,Muirhead 2002,https://pubmed.ncbi.nlm.nih.gov/11879258/,"erythromycin DDI, treatment group, D6",Sildenafil,Erythromycin,PO,PO,,Plasma,2.579306723,Approx. avg,,,2.088926174,Approx. avg,,,121,inf,121,inf,h,,,500,,mg,,"24, 36, 48, 60, 72, 84, 96, 108, 120",h,,,,Unknown,,,,Fed,,,light breakfast

```

Comment:
Compound names are ususally written starting with the capital letter; in the last 5 lines, column Grouping, it's not the case (e.g. "**e**rythromycin DDI, treatment group, D6")